__tests__/categories-router.test.js

```diff
@@ -29,6 +35,59 @@ describe('categories-router testing', () => {
                         expect(res.status).toBe(200);
                     })
         })
+        it('should return proper JSON response', () => {
+            return request(server).get('/todos/categories').set('Authorization', generatedToken)
+                .then(res => {
+                    expect(res.body).toEqual([
+                        {id: 1, category_name: 'living_room'},
+                        {id: 2, category_name: 'bedroom'},
+                        {id: 3, category_name: 'kitchen'},
+                        {id: 4, category_name: 'bathroom'}
+                      ])
+                })
+        })
+    })
+    describe('GET todos/categories/todoID', () => {
+        it('should return status 404 when invalid ID is passed in', () => {
+            return request(server).get('/todos/categories/25').set('Authorization', generatedToken)
+                .then(res => {
+                    expect(res.status).toBe(404)
+                })
+        })
+        it('should return status 200 when valid ID is passed in', () => {
+            return request(server).get('/todos/categories/2').set('Authorization', generatedToken)
+                .then(res => {
+                    expect(res.status).toBe(200)
+                })
+        })
+        it('should return proper category', () => {
+            return request(server).get('/todos/categories/2').set('Authorization', generatedToken)
+                .then(res => {
+                    expect(res.body).toEqual(["kitchen"])
+                })
+            })
+    })
+    describe('POST todos/categories/', () => {
+        it('returns 201 status on successful add', () => {
+            return request(server).post('/todos/categories/').send({todo_id: 1, category_name: 'bedroom'}).set('Authorization', generatedToken)
+                .then(res => {
+                    expect(res.status).toBe(201)
+                })
+        })
+        it('returns 500 error if missing information', () => {
+            return request(server).post('/todos/categories/').send({category_name: 'bedroom'}).set('Authorization', generatedToken)
+                .then(res => {
+                    expect(res.status).toBe(500)
+                })
+        })
+    })
+    describe('DELETE todos/categories/', () => {
+        it('returns 201 error upon successful delete' , () => {
```

Comment:
Making note of a couple inconsistencies here: 

- It says 201, but the test is checking the status to be equal to 200

- It says "error" but should just be "status"

Aside from that, all seems good

Reply:
Ahh thanks that was a failed copy an paste Ill change that